src/Card/Card.scss

```diff
@@ -1,2 +1,53 @@
 @import "variables";
 @import "~bootstrap/scss/card";
+
+/**
+ * Card Header
+ */
+
+ .pgn__card-header {
+  @extend .card-body;
+  @extend .pb-0;
+}
+
+.pgn__card-header-title {
+  @extend .m-0;
+}
+
+/**
+ * Card Section
+ */
+
+.pgn__card-section {
+  @extend .card-body;
+  @extend .py-3;
+}
+
+.pgn__card-section-actions {
+  @extend .mt-2;
+  @extend .mb-0;
+  @extend .text-right;
+}
+
+.pgn__card-section-action {
+  @extend .ml-1;
+  @extend .btn-sm;
+
+  &:first-child {
+    @extend .ml-0;
+  }
+}
+
+// .pgn__card-section-content {}
+// .pgn__card-section-title {}
+
+/**
+ * Card Divider
+ */
+
+.pgn__card-divider {
+  height: 0;
+  margin: $card-divider-margin-y 0;
```

Comment:
`$card-divider-margin-y` and `$card-divider-bg` are new variables. Should they be added to `@edx/brand`?

Reply:
No, but let's document them in the doc site. The edx brand currently attempts have all the available SASS variables  commented out, but this is done mostly because we have no documentation of what variables exist. We'll change that soon and then it'll just be a burden to update without any added value.

Reply:
Got it, this makes sense!